spring-boot-project/spring-boot-docs/src/docs/antora/modules/reference/pages/features/dev-services.adoc

```diff
@@ -402,7 +402,7 @@ You can also add javadoc:org.springframework.test.context.DynamicPropertySource[
 
 When using devtools, you can annotate beans and bean methods with javadoc:org.springframework.boot.devtools.restart.RestartScope[format=annotation].
 Such beans won't be recreated when the devtools restart the application.
-This is especially useful for Testcontainer javadoc:org.testcontainers.containers.Container[] beans, as they keep their state despite the application restart.
+This is especially useful for Testcontainers javadoc:org.testcontainers.containers.Container[] beans, as they keep their state despite the application restart.
```

Comment:
I think this was intentionally singular. If it's to change, I think it should be changed to "container".

Reply:
@wilkinsona How about "Testcontainers'" or just removing "Testcontainer"?

"container Container beans" doesn't sound right to me.

Reply:
Oops. Yeah, I think we can just remove it as we're already in a section about Testcontainers. I think "for Container beans" reads better and there's enough context to know that it's a Testcontainers container.